src/NHibernate.Test/App.config

```diff
@@ -7,7 +7,7 @@
 	</configSections>
 
 	<connectionStrings>
-		<add name="TestConnectionString" connectionString="Server=localhost\sqlexpress;Database=nhibernate;Integrated Security=SSPI" />
+		<add name="TestConnectionString" connectionString="Server=localhost;Database=nhibernate;Integrated Security=SSPI" />
```

Comment:
Please revert. You should not need to change that. Use ShowBuildMenu and setup your test configuration with it.